src/pages/quizzes.tsx

```diff
@@ -0,0 +1,224 @@
+          />
+        </QuizzesModal>
+
+        <Box px={{ base: 0, lg: 8 }} py={{ base: 0, lg: 4 }} mb={12}>
+          <Flex
+            direction={{ base: "column", lg: "row" }}
+            alignItems="start"
+            gap={{ base: 0, lg: 20 }}
+          >
+            {/* quizzes list */}
+            <Box flex={1} order={{ base: 2, lg: 1 }}>
+              <Box px={{ base: 8, lg: 0 }}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="basics" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="basics-description" />
+                </Text>
+
+                <QuizzesList
+                  content={ethereumBasicsQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Box px={{ base: 8, lg: 0 }} mb={10}>
+                <Heading
+                  fontSize={{ base: "1.75rem", lg: "2rem" }}
+                  color="body"
+                >
+                  <Translation id="using-ethereum" />
+                </Heading>
+
+                <Text mb={8} color="body">
+                  <Translation id="using-ethereum-description" />
+                </Text>
+
+                <QuizzesList
+                  content={usingEthereumQuizzes}
+                  quizHandler={setCurrentQuiz}
+                  nextHandler={setNextQuiz}
+                  modalHandler={setModalOpen}
+                />
+              </Box>
+
+              <Flex
+                direction={{ base: "column", xl: "row" }}
+                justifyContent="space-between"
+                alignItems="center"
+                bg="backgroundHighlight"
+                borderRadius={{ base: "none", lg: "lg" }}
+                border="none"
+                p={4}
+              >
+                <Stack mb={{ base: 4, xl: 0 }}>
+                  <Text
+                    align={{ base: "center", xl: "left" }}
+                    fontWeight="bold"
+                    color="body"
+                    mb={-2}
+                  >
+                    <Translation id="want-more-quizzes" />
+                  </Text>
+
+                  <Text align={{ base: "center", xl: "left" }} color="body">
+                    <Translation id="contribute" />
+                  </Text>
+                </Stack>
+
+                <ButtonLink
+                  to={"/contributing/learning-quizzes/"}
+                  variant="outline"
+                  hideArrow
+                  mt={0}
+                >
+                  <Flex alignItems="center">
+                    <Icon as={FaGithub} color="text" boxSize={6} me={2} />
+                    <Translation id="add-quiz" />
+                  </Flex>
+                </ButtonLink>
```

Comment:
@nhsz @nloureiro re: button icon bug discussion

We should use it like this now (using the `leftIcon` or `rightIcon` prop):

```suggestion
                <ButtonLink
                  to={"/contributing/learning-quizzes/"}
                  variant="outline"
                  leftIcon={<FaGithub />}
                  hideArrow
                  mt={0}
                >
                  <Translation id="add-quiz" />
                </ButtonLink>
```

Reply:
@pettinarip any way to adjust the icon size?

Reply:
Ah, in that case, you would need to use the `Icon` component as before.

```tsx
<ButtonLink
  to={"/contributing/learning-quizzes/"}
  variant="outline"
  leftIcon={<Icon as={FaGithub} boxSize={6} />}
```